test/unit/ngsiv2/HTTP_reveice_measures-test.js

```diff
@@ -79,13 +79,11 @@ describe('HTTP: Measure reception ', function() {
 
```

Comment:
It would be great to to a grep in test/unit/ngsv2/ for `/v1` to ensure that everything has been changed, as was done in the library PR. Maybe you have done it already :)

Reply:
I already did it! ;)

```
a620381@a620381-VirtualBox:~/workspace/finext/iotagent-json$ grep -Rin v1 test/unit/ngsiv2/
a620381@a620381-VirtualBox:~/workspace/finext/iotagent-json$
```

Reply:
Great! NTC